python/lsst/sims/catUtils/mixins/AstrometryMixin.py

```diff
@@ -92,6 +94,11 @@ class AstrometryGalaxies(AstrometryBase):
     This mixin contains a getter for the corrected RA and dec which ignores parallax and proper motion
```

Comment:
Yes, but because they are not specified as kwargs, parallax and proper motion are set to None.

Basically, I'm trying to say that this getter is for objects that do not have a parallax or proper motion defined.

I will try to rewrite it.